### DISCUSSION TITLE
tutorials/client-side-graphql-react/01-feature-overview-and-setup

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#147] [course feedback]
# tutorials/client-side-graphql-react/01-feature-overview-and-setup

Learn about the frontend app you'll be building throughout this Client-side GraphQL course

https://www.apollographql.com/tutorials/client-side-graphql-react/01-feature-overview-and-setup

**Comment by arrunks** [comment]
The project name is invalid, I had to replace spaces with '-' to make npm install work

**Reply by mabuyo** [reply]
Hey there @arrunks - can you clarify what you mean by "project name"? And what error you're seeing when you run `npm install`?

**Reply by arrunks** [reply]
npm ERR! Invalid name: "Odyssey Clientside GraphQL"

**Reply by mabuyo** [reply]
I can't reproduce the error but after some [digging](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#name), I see that the `name` field should be URL-friendly. We'll fix this shortly for this repo and look into our other repos. Thank you!

**Comment by edannert** [comment]
Above it says: "In a terminal window, navigate to the root of the project directory" - for someone who does not develop on a regular basis that was not clear enough and I had to think about what you mean, realising it is the "src" folder I had to go into in my terminal and run npm install and start. Once I figured it easy, but I suggest you be more specific, so people know what you are referring to by "root".
